radar/2021-01-01/docker.md

```diff
@@ -0,0 +1,7 @@
+---
+featured: false
```

Comment:
You need to copy over the full header meta data, otherwise, Docker will show up as `docker.md` in the overview.
There is no merge happening with the previous files, thats also the reason why there are a lot of broken entries in the previous tech radar edition.. 😢

Reply:
But... shouldn't that be fixed in the techradar then? :nerd_face: It seems to be a problem with existing updates already.